.travis.yml

```diff
@@ -1,18 +1,16 @@
 language: ruby
 rvm:
   - jruby
-  - 2.0.0
-  - 2.1
-  - 2.2
-  - 2.3.0
+  - 2.1.10
+  - 2.2.6
+  - 2.3.3
```

Comment:
@kyuden, I'm sure what you have is fine. I just want to verify that we weren't using 2.1 and 2.2 for a specific reason and won't have time until this weekend to do so.

Thanks again for getting this in, the only blocker I see right now is the travis versioning; so it should be merged soon.